react/week-2/lesson.md

```diff
@@ -182,7 +182,7 @@ Let's look at how a component is re-rendered ([interactive version](https://code
 function Counter(props) {
   console.log(`Rendering. props.count is ${props.likeCount}`);
 
-  return <button id="like-button">Likes: {props.likeCount}</button>;
+  return <button className="like-button">Likes: {props.likeCount}</button>;
```

Comment:
Thanks for the suggestion here. In this case, I think an `id` is more appropriate. If you look at the [linked CodeSandbox](https://codesandbox.io/s/force-component-re-rendering-llow115pll?file=/src/index.js), you'll see that I'm using this `id` to attach an event listener to.

Generally I use `id`s for attaching JS events and `class`es for styling. Hence in this case, an `id`.

Reply:
Admittedly I'm also using the `id` for styling, but I figured it was less distracting to do that than add both an `id` and a `class`.